docs/stable/cli/cli_api.md

```diff
@@ -106,16 +106,37 @@ This file can be incomplete, and missing sections will be filled in by the defau
         "metric": "concurrency",
         "target": 1,
         "min_instances": 0,
-        "max_instances": 10
+        "max_instances": 10,
+        "keep_alive": 0
     },
     "backend_config": {
         "pretrained_model_name_or_path": "facebook/opt-1.3b",
         "device_map": "auto",
-        "torch_dtype": "float16"
+        "torch_dtype": "float16",
+        "hf_model_class": "AutoModelForCausalLM"
     }
 }
 ```
 
+Below is a description of all the fields in config.json.
+
+| Field | Description |
+| ----- | ----------- |
+| model | This should be a HuggingFace model name, used to identify model instance. |
+| backend | Inference engine, support `transformers` and `vllm` now. |
+| num_gpus | Number of GPUs used to deploy a model instance. |
+| auto_scaling_config | Config about auto scaling. |
+| auto_scaling_config.metric | Metic used to decide whether to scale up or down. |
```

Comment:
Hi, there's a typo—it should be `Metric`. We also have a pre-commit tool mentioned in `CONTRIBUTING.md`. You can use it before committing to automatically check for typos.

Reply:
Thanks for your reminder, I have fixed it in the new commit. By the way, I used `pre-commit run -a` but it did not report that problem.